TestStepVerifyPreInstallNoKanikoLazyCreate fails when running test-slow-integration locally

TestStepVerifyPreInstallNoKanikoLazyCreate of `step_verify_preinstall_integration_test.go` fails when trying to run `make test-slow-integration` locally.

The problem is that it tries to clone into an existing ~/.jx directory. 

```
--- FAIL: TestStepVerifyPreInstallNoKanikoLazyCreate (4.39s)
    step_verify_preinstall_integration_test.go:72: 
        	Error Trace:	step_verify_preinstall_integration_test.go:72
        	Error:      	Received unexpected error:
        	            	non-fast-forward update
        	            	pulling the latest
        	            	github.com/jenkins-x/jx/pkg/cmd/opts.(*CommonOptions).CloneJXVersionsRepo
        	            		/Users/hardy/work/go/src/github.com/jenkins-x/jx/pkg/cmd/opts/helm.go:529
        	            	github.com/jenkins-x/jx/pkg/cmd/opts.(*CommonOptions).CreateVersionResolver
        	            		/Users/hardy/work/go/src/github.com/jenkins-x/jx/pkg/cmd/opts/version.go:16
        	            	github.com/jenkins-x/jx/pkg/cmd/step/verify.(*StepVerifyPackagesOptions).Run
        	            		/Users/hardy/work/go/src/github.com/jenkins-x/jx/pkg/cmd/step/verify/step_verify_packages.go:90
        	            	github.com/jenkins-x/jx/pkg/cmd/step/verify.(*StepVerifyPreInstallOptions).Run
        	            		/Users/hardy/work/go/src/github.com/jenkins-x/jx/pkg/cmd/step/verify/step_verify_preinstall.go:150
        	            	github.com/jenkins-x/jx/pkg/cmd/step/verify_test.TestStepVerifyPreInstallNoKanikoLazyCreate
        	            		/Users/hardy/work/go/src/github.com/jenkins-x/jx/pkg/cmd/step/verify/step_verify_preinstall_integration_test.go:71
        	            	testing.tRunner
        	            		/Users/hardy/homebrew/Cellar/go/1.12.4/libexec/src/testing/testing.go:865
        	            	runtime.goexit
        	            		/Users/hardy/homebrew/Cellar/go/1.12.4/libexec/src/runtime/asm_amd64.s:1337
        	            	failed to create version resolver
        	            	github.com/jenkins-x/jx/pkg/cmd/step/verify.(*StepVerifyPackagesOptions).Run
        	            		/Users/hardy/work/go/src/github.com/jenkins-x/jx/pkg/cmd/step/verify/step_verify_packages.go:92
        	            	github.com/jenkins-x/jx/pkg/cmd/step/verify.(*StepVerifyPreInstallOptions).Run
        	            		/Users/hardy/work/go/src/github.com/jenkins-x/jx/pkg/cmd/step/verify/step_verify_preinstall.go:150
        	            	github.com/jenkins-x/jx/pkg/cmd/step/verify_test.TestStepVerifyPreInstallNoKanikoLazyCreate
        	            		/Users/hardy/work/go/src/github.com/jenkins-x/jx/pkg/cmd/step/verify/step_verify_preinstall_integration_test.go:71
        	            	testing.tRunner
        	            		/Users/hardy/homebrew/Cellar/go/1.12.4/libexec/src/testing/testing.go:865
        	            	runtime.goexit
        	            		/Users/hardy/homebrew/Cellar/go/1.12.4/libexec/src/runtime/asm_amd64.s:1337
        	Test:       	TestStepVerifyPreInstallNoKanikoLazyCreate
        	Messages:   	the command should not have failed as we should have lazily created the deploy namespace
```

It might work to set JX_HOME for this test to foce the clone of the version stream into a tmp directory.

--- a/pkg/cmd/step/verify/step_verify_preinstall_integration_test.go
+++ b/pkg/cmd/step/verify/step_verify_preinstall_integration_test.go
@@ -61,6 +61,19 @@ func TestStepVerifyPreInstallNoKanikoNoLazyCreate(t *testing.T) {
 }
 
 func TestStepVerifyPreInstallNoKanikoLazyCreate(t *testing.T) {
+	origJxHome := os.Getenv("JX_HOME")
+
+	tmpJxHome, err := ioutil.TempDir("", "jx-test-TestStepVerifyPreInstallNoKanikoLazyCreate")
+	assert.NoError(t, err)
+
+	err = os.Setenv("JX_HOME", tmpJxHome)
+	assert.NoError(t, err)
+
+	defer func() {
+		_ = os.RemoveAll(tmpJxHome)
+		err = os.Setenv("JX_HOME", origJxHome)
+	}()
+
 	options := createTestStepVerifyPreInstallOptions(filepath.Join("test_data", "preinstall", "no_kaniko_or_terraform"))
 
 	_, origNamespace, err := options.KubeClientAndDevNamespace()

